### ISSUE TITLE
Page functions not working

### ISSUE THREAD

**Issue by Elvis-Besingi** [gctools-outilsgc/gcconnex#2609]
Hello Team,

The users of this group have issues modifying their community page.

Here is a link to the page 

https://gccollab.ca/file/group/3603354/all

- Unable to select folders
- Unable to use the new file folder bottom
- Unable to search new group members to remove specific people from group.
- User is the group owner

https://gccollab.ca/file/group/3603354/all#15072279

![image](https://github.com/gctools-outilsgc/gcconnex/assets/123134358/dc3cd280-c05c-44b4-abaf-c954d6baac43)

Thank you for your usual help.

**Comment by Phanoix** [comment]
tl;dr workaround: your language setting / cookie broke, hit the langauge toggle to fix it
I also figured out a bit more about this after looking into it again so I'll see if we can get this bug fixed completely this time

The whole story:
This is related to a language cookie bug that that's been around for a while and stems from the fact that we have one language cookie shared between the gccollab tools, it looks like under specific circumstances (the language cookie is expired or otherwise not already set) the language cookie can be set to "undefined" by the wiki (the relevant script is the top bit of [common.js](https://wiki.gccollab.ca/MediaWiki:Common.js)) but it only causes issues if you then go to gccollab before that cookie expires the next day.

Now what this does to gccollab is very wide-spread, including breaking all text that's using the bilingual content functionality and a lot of javascript-based functionality such as that select all toggle and the new folder popup.

Fixing it when it happens to you is a simple as hitting the language toggle, though now that I think I finally figured out the whole process of how the bug happens I should be able to do something about it that's more resilient than the previous attempts to fix it.

**Comment by Phanoix** [comment]
After some more investigating the language cookie script wasn't doing much more than breaking the cookie occasionally, and even if it was working properly there are some issues, for example 'en-ca' would also be a problem. I've removed the script so there shouldn't be anything breaking the cookie anymore.
